src/main/java/com/o19s/es/ltr/query/StoredLtrQueryBuilder.java

```diff
@@ -67,9 +68,17 @@
         declareStandardFields(PARSER);
     }
 
-    public StoredLtrQueryBuilder() {}
-    public StoredLtrQueryBuilder(StreamInput input) throws IOException {
+    public StoredLtrQueryBuilder() {
```

Comment:
I see what you mean, I'll move this behaviors to the test, it'll me give a chance to throw some hints on how to use it in integration tests.